templates/SimpleInventory/edit-board.html

```diff
@@ -18,28 +18,55 @@
 
 {% block content %}
 <div id="formCard" class="form-card mdl-card mdl-cell--8-col mdl-cell--2-offset-desktop mdl-shadow--2dp mdl-grid">
-    <div class="mdl-card__title">
-        <h2 class="mdl-card__title-text">Edit Board</h2>
-    </div>
-    <div class="mdl-card__supporting-text">
-        <form method="post" action="{% url 'api_v1_board_update' board_id %}" id="editBoardForm" class="mdl-cell--2-offset-desktop mdl-cell--8-col">
-            {% csrf_token %}
-            {{ form.active.as_hidden }}
-            <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label mdl-cell--12-col mdl-cell--bottom">
-                {{ form.name }}
-                <label class="mdl-textfield__label" for="{{ form.name.id_for_label }}">Name</label>
-            </div>
-            <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label mdl-cell--12-col mdl-cell--bottom">
-                {{ form.desc }}
-                <label class="mdl-textfield__label" for="{{ form.desc.id_for_label }}">Description</label>
-            </div>
-            <div class="mdl-card__actions">
-                <div class="mdl-layout-spacer"></div>
-                <button class="mdl-button mdl-js-button mdl-button--raised mdl-js-ripple-effect mdl-button--primary">
-                    Edit <i class="material-icons right">done</i>
-                </button>
-            </div>
-        </form>
-    </div>
+  <div class="mdl-card__title">
+    <h2 class="mdl-card__title-text">Edit Board</h2>
+  </div>
+  <div class="mdl-card__supporting-text">
+    <form method="post" action="{% url 'api_v1_board_update' board_id %}" id="editBoardForm" class="mdl-cell--2-offset-desktop mdl-cell--8-col">
+      {% csrf_token %}
+      {{ form.active.as_hidden }}
+      <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label mdl-cell--12-col mdl-cell--bottom">
+        {{ form.name }}
+        <label class="mdl-textfield__label" for="{{ form.name.id_for_label }}">Name</label>
+      </div>
+      <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label mdl-cell--12-col mdl-cell--bottom">
+        {{ form.desc }}
+        <label class="mdl-textfield__label" for="{{ form.desc.id_for_label }}">Description</label>
+      </div>
+
+      <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label">
+        {{ form.readGroup }}
+        <label for="{{ form.readGroup.id_for_label }}" class="mdl-textfield__label">Read Group</label>
```

Comment:
Waterflow permissions for now. ie add implies view...We can reevaluate later if that is okay...